Notifications related crash in full flavor

<!-- Please search the issue, if there is one with your issue -->

````
Fatal Exception: java.lang.RuntimeException: Unable to instantiate service org.openhab.habdroid.core.FcmMessageListenerService: java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.Object android.content.Context.getSystemService(java.lang.String)' on a null object reference
       at android.app.ActivityThread.handleCreateService(ActivityThread.java:3745)
       at android.app.ActivityThread.access$1400(ActivityThread.java:238)
       at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1803)
       at android.os.Handler.dispatchMessage(Handler.java:106)
       at android.os.Looper.loop(Looper.java:214)
       at android.app.ActivityThread.main(ActivityThread.java:7073)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:494)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:965)
Caused by java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.Object android.content.Context.getSystemService(java.lang.String)' on a null object reference
       at android.content.ContextWrapper.getSystemService(ContextWrapper.java:741)
       at org.openhab.habdroid.core.NotificationHelper.<init>(NotificationHelper.kt:41)
       at org.openhab.habdroid.core.FcmMessageListenerService.<init>(FcmMessageListenerService.kt:23)
       at java.lang.Class.newInstance(Class.java)
       at android.app.AppComponentFactory.instantiateService(AppComponentFactory.java:103)
       at androidx.core.app.CoreComponentFactory.instantiateService(CoreComponentFactory.java:75)
       at android.app.ActivityThread.handleCreateService(ActivityThread.java:3740)
       at android.app.ActivityThread.access$1400(ActivityThread.java:238)
       at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1803)
       at android.os.Handler.dispatchMessage(Handler.java:106)
       at android.os.Looper.loop(Looper.java:214)
       at android.app.ActivityThread.main(ActivityThread.java:7073)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:494)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:965)
````


### Environment data
#### Client
* Android version: Multiple
* Device model: Multiple
* App version <!-- Can be found under "About" -->: 2.13.1-beta 
* Build flavor <!-- Play Store and GitHub = "Full", F-Droid = "Foss". If self-built and you haven't changed the flavor, it's Foss -->: Full
* Device language: Multiple

I experience same issue, beta app instantly crashing on reception of a OH notification message, regardless of source OH server or test message on myopenhab.org and regardless of local (direct) or remote (Cloud) connection. Happens since "new" version of beta, 2.13.x. Running in parallel to released OH Andoid app, which receives messages without any issues. I re-established connection of both app versions to myopenhab just yesterday, to exclude that part from the issue. Hoping that this log section helps, identified by crash timestamp:
```
--------- beginning of main
06-24 09:00:49.099  8890  8890 D ViewRootImpl@75b56a4[LogActivity]: MSG_WINDOW_FOCUS_CHANGED 0 1
06-24 09:00:49.099  8890  8890 D InputMethodManager: prepareNavigationBarInfo() DecorView@dbb30a8[LogActivity]
06-24 09:00:49.099  8890  8890 D InputMethodManager: getNavigationBarColor() -16777216
06-24 09:00:56.308  8890  8890 D ViewRootImpl@75b56a4[LogActivity]: MSG_WINDOW_FOCUS_CHANGED 1 1
06-24 09:00:56.308  8890  8890 D InputMethodManager: prepareNavigationBarInfo() DecorView@dbb30a8[LogActivity]
06-24 09:00:56.308  8890  8890 D InputMethodManager: getNavigationBarColor() -16777216
06-24 09:01:06.351  8890  8890 D AndroidRuntime: Shutting down VM
--------- beginning of crash
06-24 09:01:06.352  8890  8890 E AndroidRuntime: FATAL EXCEPTION: main
06-24 09:01:06.352  8890  8890 E AndroidRuntime: Process: org.openhab.habdroid.beta, PID: 8890
06-24 09:01:06.352  8890  8890 E AndroidRuntime: java.lang.RuntimeException: Unable to instantiate service org.openhab.habdroid.core.FcmMessageListenerService: java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.Object android.content.Context.getSystemService(java.lang.String)' on a null object reference
06-24 09:01:06.352  8890  8890 E AndroidRuntime: at android.app.ActivityThread.handleCreateService(ActivityThread.java:4269)
06-24 09:01:06.352  8890  8890 E AndroidRuntime: at android.app.ActivityThread.access$1800(ActivityThread.java:272)
06-24 09:01:06.352  8890  8890 E AndroidRuntime: at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2071)
06-24 09:01:06.352  8890  8890 E AndroidRuntime: at android.os.Handler.dispatchMessage(Handler.java:107)
06-24 09:01:06.352  8890  8890 E AndroidRuntime: at android.os.Looper.loop(Looper.java:237)
06-24 09:01:06.352  8890  8890 E AndroidRuntime: at android.app.ActivityThread.main(ActivityThread.java:8016)
06-24 09:01:06.352  8890  8890 E AndroidRuntime: at java.lang.reflect.Method.invoke(Native Method)
06-24 09:01:06.352  8890  8890 E AndroidRuntime: at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
06-24 09:01:06.352  8890  8890 E AndroidRuntime: at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1076)
06-24 09:01:06.352  8890  8890 E AndroidRuntime: Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.Object android.content.Context.getSystemService(java.lang.String)' on a null object reference
06-24 09:01:06.352  8890  8890 E AndroidRuntime: at android.content.ContextWrapper.getSystemService(ContextWrapper.java:766)
06-24 09:01:06.352  8890  8890 E AndroidRuntime: at org.openhab.habdroid.core.NotificationHelper.<init>(NotificationHelper.kt:41)
06-24 09:01:06.352  8890  8890 E AndroidRuntime: at org.openhab.habdroid.core.FcmMessageListenerService.<init>(FcmMessageListenerService.kt:23)
06-24 09:01:06.352  8890  8890 E AndroidRuntime: at java.lang.Class.newInstance(Native Method)
06-24 09:01:06.352  8890  8890 E AndroidRuntime: at android.app.AppComponentFactory.instantiateService(AppComponentFactory.java:129)
06-24 09:01:06.352  8890  8890 E AndroidRuntime: at androidx.core.app.CoreComponentFactory.instantiateService(CoreComponentFactory.java:75)
06-24 09:01:06.352  8890  8890 E AndroidRuntime: at android.app.ActivityThread.handleCreateService(ActivityThread.java:4264)
06-24 09:01:06.352  8890  8890 E AndroidRuntime: ... 8 more
06-24 09:01:06.423  8890  8935 I System.out: (HTTPLog)-Static: isSBSettingEnabled false
06-24 09:01:06.423  8890  8935 I System.out: (HTTPLog)-Static: isSBSettingEnabled false
06-24 09:01:07.205  8890  8935 I CrashlyticsCore: Crashlytics report upload complete: 5EF2FA070322-0001-22BA-3860A4FE6C64
```
Let me know section of interest from app starting log, like this, 1s after app restart:
```
06-24 09:01:32.221  9572  9572 D MainActivity: onCloudConnectionChanged()
06-24 09:01:32.230  9572  9703 D FcmRegistrationService: Register device at openHAB-cloud with URL: addAndroidRegistration?deviceId=076679dd4889f6ff-beta&deviceModel=Samsung+SM-G973F+%28Beta%29&regId=eoAyzoANRhG2k8J...
06-24 09:01:32.230  9572  9685 I okhttp.OkHttpClient: --> GET https://<openhab-remote-address>/addAndroidRegistration?deviceId=076679dd4889f6ff-beta&deviceModel=Samsung+SM-G973F+%28Beta%29&regId=eoAyzoANRhG2k8J...
```

--- a/mobile/src/full/java/org/openhab/habdroid/core/FcmMessageListenerService.kt
+++ b/mobile/src/full/java/org/openhab/habdroid/core/FcmMessageListenerService.kt
@@ -20,7 +20,12 @@ import org.openhab.habdroid.model.CloudNotification
 import org.openhab.habdroid.model.toOH2IconResource
 
 class FcmMessageListenerService : FirebaseMessagingService() {
-    private val notifHelper = NotificationHelper(this)
+    private lateinit var notifHelper: NotificationHelper
+
+    override fun onCreate() {
+        super.onCreate()
+        notifHelper = NotificationHelper(this)
+    }
 
     override fun onNewToken(token: String) {
         super.onNewToken(token)

